src/javascript/app/pages/user/metatrader/metatrader_terminal.js

```diff
@@ -0,0 +1,52 @@
+const loadScript  = require('scriptjs');
+const DerivBanner = require('../../../common/deriv_banner');
+const getLanguage = require('../../../../_common/language').get;
+
+const MetaTraderTerminal = (() => {
+    let is_loaded;
+
+    const onLoad = () => {
+        const curr_language = getLanguage().toLowerCase();
+        const urlParams = new URLSearchParams(window.location.search);
+        const login = urlParams.get('login');
+        const servers = urlParams.get('servers');
+        const trade_server = urlParams.get('trade_server');
+
+        DerivBanner.onLoad();
+        if (!is_loaded) {
+            loadScript.get('https://trade.mql5.com/trade/widget.js', () => {
+                loadTerminal(curr_language, login, servers, trade_server);
+                is_loaded = true;
+            });
+        } else {
+            loadTerminal('en', login, servers, trade_server);
```

Comment:
why are we forcing english after first load?

Reply:
My bad, I forgot to update.